client/src/featureform/sqlite_metadata.py

```diff
@@ -194,13 +193,19 @@ def getVariantResource(self, type, column, resource):
         variant_table_query = "SELECT * FROM " + type + " WHERE " + column + "='" + resource + "';"
         variant_data = self.__conn.execute(variant_table_query)
         self.__conn.commit()
+        variant_data_list = variant_data.fetchall()
+        if len(variant_data_list) == 0:
+          raise ValueError(type + " with " + column + " : " + resource + " not found")
         return variant_data.fetchall()
 
     def getNameVariant(self, type, column1, resource1, column2, resource2):
```

Comment:
These need to be changed to snake case, but I think thats part of the refactor task